process-engine/bpmn-studio
# ✨ Add Message Selection for Message Tasks

## What did you change?

This PR adds the message selection for the send/receive task.

Fixes: #584 

## How can others test the changes?

- Open up the BPMN-Studio
- Go to the Detail View of any diagram
- Click on a send/receive task
- Notice the message selection in the property panel

## PR-Checklist

Please check the boxes in this list after submitting your PR:
